### ISSUE TITLE
GPU Load Imbalance

### ISSUE THREAD

**Issue by ChongZhangZC** [JDAI-CV/fast-reid#130]
Is there any other way to solve the problem of GPU load imbalance?  It is not easy to modify the code directly with Pytorch-encoding and distributedDataparallel.
I can't increase the batchsize more than 64 on 4 gpus(12G) because of OOM error on one of them.

**Comment by L1aoXingyu** [comment]
We are trying to use DDP instead of DP, but it still needs some efforts.
There is something to consider such as hard mining in triplet loss between different processes.
But we have finished 60% of the work, maybe we will merge it in fastreid in a week.

**Comment by ChongZhangZC** [comment]
@L1aoXingyu Excellent! Thank you very much, looking forward to the updated fastreid!

**Comment by L1aoXingyu** [comment]
@ChongZhangZC 
Hi, we have updated DDP training, you try it.
